index.js

```diff
@@ -419,6 +419,52 @@
     return f(y, x);
   });
 
+  //# lift :: Applicative f => (a -> b) -> f a -> f b
```

Comment:
In fact, am I correct in saying that `lift2` and `lift3` require [Apply][1] but `lift` merely requires [Functor][2]? This suggests:

```haskell
lift  :: Functor f => (a -> b) -> f a -> f b
lift2 :: Apply f => (a -> b -> c) -> f a -> f b -> f c
lift3 :: Apply f => (a -> b -> c -> d) -> f a -> f b -> f c -> f d
```


[1]: https://github.com/fantasyland/fantasy-land#apply
[2]: https://github.com/fantasyland/fantasy-land#functor